docs/agent-recipes.md

```
@@ -0,0 +1,296 @@
+    encoded_query="active=true",
+    aggregate="count",
+    group_by="assignment_group"
+)
+
+# report["data"] will contain list of {assignment_group: "sys_id", count: "42"}
+```
+**Notes:** Aggregate queries return the raw sys_id of the group. Call `query` with `sys_id` mode on `sys_user_group` to resolve the top group's name if needed.
+
+---
+
+### 11. Inspect Flow Designer artifacts
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)